source/CppTestSrv/SERVER.IDL

```diff
@@ -0,0 +1,72 @@
+//
+// Server.idl
+//
+// This file will be processed by the MIDL compiler to
+// produce the type library (server.tlb) and marshaling code.
+//
+
+import "oaidl.idl" ;
+
+typedef [uuid(00FABB0F-5691-41A6-B7C1-11606671F8E5)]
+struct T_TEST_RECORD {
+	BSTR question ;
+	long answer ;
+	VARIANT_BOOL needs_clarification ;
+} T_TEST_RECORD ;
```

Comment:
This structure should also be named in accordance with its responsibility to be used in "RecordParamTest".
```suggestion
struct StructRecordParamTest {
	BSTR question ;
	long answer ;
	VARIANT_BOOL needs_clarification ;
} StructRecordParamTest ;
```